### PR TITLE
Contributing.md and DCO

### PR DESCRIPTION
**Pull request by clux**
Based on simpler requirements and discussion after https://github.com/kube-rs/kube-rs/issues/585#issuecomment-892070891

### CODE REVIEW

**Comment by clux** [comment]
Ok, the DCO does take a bit of work for users the first time. The instructions for the last commit is clear:

```sh
git commit --amend --signoff
git push -f
```

but if you have more than one commit you actually have to:

```sh
git rebase --exec 'git commit --amend --no-edit -n --signoff' -i origin/master
git push -f
```

File: LICENSE [hunk 1]
```diff
@@ -187,7 +187,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright 2014 The Kubernetes Authors.
+   Copyright 2018 The kube-rs Authors.
```
Comment (teozkr) [inline LICENSE:190]:
```suggestion
   Copyright 2018-2021 The kube-rs Authors.
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+[here](https://developercertificate.org/)
+
+## Pull Request Management
+
+All code that is contributed to kube-rs must go through the Pull Request (PR)
+process. To contribute a PR, fork this project, create a new branch, make
+changes on that branch, and then use GitHub to open a pull request with your
+changes.
+
+Every PR must be reviewed by at least one Core Maintainer of the project. Once
```
Comment (teozkr) [inline CONTRIBUTING.md:19]: Nit: Might be worth defining a list of Core Maintainers somewhere (codeowners file? org members?)

In the latter case, you're still listed as private over at https://github.com/orgs/kube-rs/people.

**Review by teozkr — APPROVED** [review]

File: LICENSE (hunk 1 above)
Comment (clux) [reply LICENSE:190]: bumping years in copyright files isn't actually a legally viable nor necessary thing to do: https://matija.suklje.name/how-and-why-to-properly-write-copyright-statements-in-your-code#why-keep-the-year

File: CONTRIBUTING.md (hunk 2 above)
Comment (clux) [reply CONTRIBUTING.md:19]: I have updated my membership to public. That was an accident.
Also I have renamed the owners team to `core-maintainers`: https://github.com/orgs/kube-rs/teams/core-maintainers

I'll add a link to it.

File: LICENSE (hunk 1 above)
Comment (clux) [reply LICENSE:190]: you could also go the [client-go way](https://github.com/kubernetes/client-go/blob/master/LICENSE#L190)

**Review by kazk — APPROVED** [review]

File: CONTRIBUTING.md (hunk 2 above)
Comment (teozkr) [reply CONTRIBUTING.md]: Doesn't look like the team is public, FWIW.

Comment (clux) [reply CONTRIBUTING.md]: It is set to visible, which is the highest setting. But apparently teams are only visible to org members, so may need to do something else there.

**Pull request merged**